srcpkgs/glab/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'glab'
 pkgname=glab
-version=1.50.0
+version=1.53.0
```

Comment:
guess the glab update was by accident?

Reply:
damn it, I'll rebased, tnx for the heads up